runtime/interpreter/accountkeys.go

```diff
@@ -37,12 +37,18 @@ func NewAuthAccountKeysValue(
 	addFunction FunctionValue,
 	getFunction FunctionValue,
 	revokeFunction FunctionValue,
+	forEachFunction FunctionValue,
+	countFunction FunctionValue,
 ) Value {
 
 	fields := map[string]Value{
 		sema.AccountKeysAddFunctionName:    addFunction,
 		sema.AccountKeysGetFunctionName:    getFunction,
 		sema.AccountKeysRevokeFunctionName: revokeFunction,
+
+		// TODO add back after flow-go is updated
```

Comment:
The update path is that Cadence introduces new requirements (functions in the runtime interface), and embedders, like flow-go, need to implement the new functions. You can uncomment the lines here and below